hardhat.config.js

```diff
@@ -40,12 +40,14 @@ module.exports = {
       ...sharedNetworkConfig,
       blockGasLimit: 100000000,
       gas: 2000000,
-      saveDeployments: true
+      saveDeployments: true,
     },
     hardhat: {
-      blockGasLimit: 100000000,
-      gas: 2000000,
-      saveDeployments: true
+      initialBaseFeePerGas: 0,
+      allowUnlimitedContractSize: true,
+      blockGasLimit: 10000000000000,
+      gas: 200000000000,
+      saveDeployments: true,
```

Comment:
I deleted the `allowUnlimitedContractSize` variable from it. I got the amount of `blockGasLimit` and `gas` from Konstantin on encountering an issue when deploying the bundled tx of the wrapper. It got reverted and the issue was to little gas. So I'm not sure if we would encounter issues with that on the mainnet. How would you go about this?

Reply:
For `blockGasLimit` I would research the current block gas limit on the mainnet. I think it's around 12.5 million gas. `gas` seems to be the pre-defined gas limit per transaction. Setting this one up doesn't pose a problem I believe -  you can also use "auto". `allowUnlimitedContractSize` I would leave out, if it works without. If it doesn't work without this flag we'll need to research a solution how we can reduce contract size for the actual deployment.

Reply:
I read the EIP2535 document. Just curious how this would fit into what we're doing here? If I understand correctly then the Multi-Facet Proxy would be like a center contract, that accesses external functions from other contracts by adding them into a mapping, that can be accessed by the msg.sender.

In our case, we're using more gas than usual because we're bundling a lot of transactions to be able to vote on only one transaction through the Gnosis UI. So the solution of Multi-Facet Proxy wouldn't fit into this I think. Or am I missing something? Also, I didn't think you meant for us to implement it, only as a reference. But I would still be curious about how this would fit our situation.

Reply:
This is about max. contract size and not for the gas limit (see point 2 in screenshot):

<img width="1662" alt="Bildschirmfoto 2021-08-19 um 14 29 41" src="https://user-images.githubusercontent.com/48454910/130068513-ea2febb3-a54e-446d-b54c-24b7039ab3ea.png">

Reply:
And yes, just as a (maybe interesting) side info :)